README.md

```diff
@@ -18,12 +18,34 @@ jobs:
   lint:
     runs-on: ubuntu-latest
 
+    permissions:
+      # required for all workflows
+      security-events: write
```

Comment:
Is this enough here, or will the app installations have to start using GITHUB_TOKEN?
If yes, I'd probably prefer to have this on by default, but with opt-out knob.

Reply:
I mean, we should have "sarif data upload opt-out" knob.  I suppose that without $GITHUB_TOKEN, even though we have that metadata specified - the action just continues working the old way.

Reply:
The `security-events: write` is required by [`github/codeql-action/upload-sarif`](https://github.com/github/codeql-action/blob/main/upload-sarif/action.yml) GitHub Action. In some repositories, it will work without this permission because they explicitly enable it in settings.

![Screenshot from 2023-01-30 09-55-52](https://user-images.githubusercontent.com/2879818/215431511-51504151-aa14-4d92-956b-860f20541425.png)

[`fedora-copr/vcs-diff-lint-action`](https://github.com/fedora-copr/vcs-diff-lint-action) stays the same, it doesn't require `GITHUB_TOKEN` it just produces SARIF file. and then users can use it however they want (e.g. upload it to GitHub as shown in the `README` example).

Reply:
nice!